app/controllers/posts_controller.rb

```diff
@@ -1,20 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @current_user = current_user
     @user = User.find(params[:user_id])
-
-    redirect_to root_path if @user.nil?
-    @posts = @user.posts
+    @posts = Post.includes(:comments).where(author_id: @user.id)
   end
 
```

Comment:
![image](https://github.com/Microverse-Fullstack-Program/rails-blog-app-v2/assets/93039359/8ca03efe-7d32-4176-a706-623373da27ed)

- [ ] It would be nice if you fix your LinkedIn handle because it is showing me 404 error 😨 Kindly fix it to make your readme descriptive and professional 👍